src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -2914,7 +2914,7 @@ abbrev zeroExtend_truncate_succ_eq_zeroExtend_truncate_or_twoPow_of_getLsbD_true
 @[deprecated and_one_eq_setWidth_ofBool_getLsbD (since := "2024-09-18")]
 abbrev and_one_eq_zeroExtend_ofBool_getLsbD := @and_one_eq_setWidth_ofBool_getLsbD
 
-@[deprecated sshiftRight_msb_eq_msb (since := "2024-10-03")]
-abbrev msb_sshiftRight := @sshiftRight_msb_eq_msb
+@[deprecated msb_sshiftRight (since := "2024-10-03")]
```

Comment:
Does this not deprecate `msb_sshiftRight`?

Reply:
not AFAIU from the docs: 
The attribute @[deprecated] on a declaration indicates that the declaration is discouraged for use in new code, and/or should be migrated away from in existing code. It may be removed in a future version of the library.
@[deprecated myBetterDef] means that myBetterDef is the suggested replacement.
@[deprecated myBetterDef "use myBetterDef instead"] allows customizing the deprecation message.
@[deprecated (since := "2024-04-21")] records when the deprecation was first applied.

Reply:
Right.